crates/driver/src/infra/simulator/mod.rs

```diff
@@ -26,16 +30,20 @@ impl Simulator {
         Self {
             inner: Inner::Ethereum(eth),
             disable_access_lists: false,
+            disable_gas: None,
         }
     }
 
     /// Disable access list simulation. Some environments, such as less popular
     /// blockchains, don't support access list simulation.
-    pub fn disable_access_lists(self) -> Self {
-        Self {
-            disable_access_lists: true,
-            ..self
-        }
+    pub fn disable_access_lists(&mut self) {
+        self.disable_access_lists = true;
+    }
+
+    /// Disable gas simulation. Useful for testing, but shouldn't be used in
+    /// production since it will cause the driver to return invalid scores.
+    pub fn disable_gas(&mut self, fixed_gas: eth::Gas) {
```

Comment:
I think this _might_ also be useful when manually running. The main reason is because nodes tend to give more details in the response of `eth_sendTransaction` than they do in the response of `eth_call`. In particular I've noticed that the `eth_call` on geth doesn't return the revert message. Also, it's easier to inspect and debug published txs.